meeting_notes/20180606_website-planning-meeting-notes.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Website Meeting Notes - June 6, 2018
+location: Toronto Reference Library
+attendees: 6
+date: 2018-06-06
+startTime: 18:00
+endTime: 16:30
+---
+
+## Attendees
+
+Kumash
+darkdrgn2k 
+Erica
+Hank
+benhylau
+garry
+
+## Website Recap
+
+- Project room is: `#design:tomesh.net`
+- GitHub repository is: https://github.com/tomeshnet/tomesh.net
+- Meeting is a follow-up to the planning session held on May 17, 2018 at HackLab.TO. [Meeting notes](./20180517_planning-meeting-notes.md)
+- 6 month period to plan out website improvements and updates
+- Open issues for follow-up in the repository's [issues section](https://github.com/tomeshnet/tomesh.net/issues)
+
+## Planning Notes
+
+### Project Onboarding
+
+- Agreed that having a _Projects_ page will help mitigate ambiguity on how one could contribute
+- Chat seems intimidating as an initial first step
+- Mailing list for those who want to stay up to date
+- _Get Involved_ page should be revised to clearly identify chat/events as places to start from
+- Homepage:
+    - Updating our tagline
+    - Renaming _Virtual Mesh_, _Install Parties_, to social nights or meet ups
+- Emphasizing meet-up style events moving forward
+
+### Wekan Deprecation
+
+- Consensed on phasing out the Wekan board as part of the project onboarding experience.
+- Experience is confusing
+- Opting to keep with GitHub for project comms and organizing
+- Lacking features that would make it easier for the group
+
+### New Projects Page
+
+- Moving forward on the creation of a Projects page
+- benhylau to start writing content for each active project
+- Initial list of projects:
+    - Mesh Orange
+    - _Building the Peer-to-Peer Internet_ Workshop
+    - IPFS streaming
+    - `prototype-cjdns-pi`
+    - Website
+- Ideas on content to attach to each project on page:
+    - Matrix project channel
+    - Categorizing icons
+    - Skill cards
+    - State of the project
+- Investigate adding _Build a Node_ in this page
```

Comment:
I thought this would become part of the prototype project and not be directly featured.

Reply:
Yeah, sounds right. Updated.